src/plugins/data/public/ui/saved_query_flyouts/open_saved_query_flyout.tsx

```diff
@@ -0,0 +1,206 @@
+
+  return (
+    <EuiFlyout onClose={onClose} hideCloseButton>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle>
+          <h3>Saved queries</h3>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiSearchBar
+          box={{
+            placeholder: 'Search saved query',
+            incremental: true,
+            schema,
+          }}
+          filters={[
+            {
+              type: 'field_value_selection',
+              field: 'language',
+              name: 'Query language',
+              multiSelect: 'or',
+              options: Array.from(queryLanguageSet).map((language) => ({
+                value: language,
+                view: language.toUpperCase(),
+              })),
+            },
+          ]}
+          onChange={onChange}
+        />
+        <EuiSpacer />
+        {queriesOnCurrentPage.length > 0 ? (
+          queriesOnCurrentPage.map((query) => (
+            <SavedQueryCard
+              key={query.id}
+              savedQuery={queryMap[query.id].savedQuery}
+              selectedQuery={selectedQuery}
+              onSelect={setSelectedQuery}
+              onRunPreview={onQueryOpen}
+              onClose={onClose}
+            />
+          ))
+        ) : (
+          <EuiEmptyPrompt content={'No saved query present'} />
+        )}
+        <EuiSpacer />
+        <EuiTablePagination
+          itemsPerPageOptions={[5, 10, 20]}
+          itemsPerPage={itemsPerPage}
+          activePage={pager.current.getCurrentPageIndex()}
+          pageCount={pager.current.getTotalPages()}
+          onChangeItemsPerPage={(pageSize) => {
+            pager.current.setItemsPerPage(pageSize);
+            setItemsPerPage(pageSize);
+            setFirstItemIndex(pager.current.getFirstItemIndex());
+            setLastItemIndex(pager.current.getLastItemIndex());
+          }}
+          onChangePage={(pageIndex) => {
+            pager.current.goToPageIndex(pageIndex);
+            setFirstItemIndex(pager.current.getFirstItemIndex());
+            setLastItemIndex(pager.current.getLastItemIndex());
+          }}
+        />
+      </EuiFlyoutBody>
+      <EuiFlyoutFooter>
+        <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty iconType={'cross'} color="danger" iconSide="left" onClick={onClose}>
+              Cancel
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup alignItems="center">
+              {/* <EuiFlexItem grow={false}>
+                <EuiCheckbox
+                  id="run-saved-query-automatically"
+                  checked={shouldRunOnOpen}
+                  onChange={(event) => {
+                    setShouldRunOnOpen(event.target.checked);
+                  }}
+                  label={'Run automatically'}
+                />
+              </EuiFlexItem> */}
```

Comment:
This commented code is supposed to be uncommented once we have the logic in place to control running of query on open

Reply:
Ideally we could just add this later, but if we're including it, can we leave a comment in the code for why its commented out?

Reply:
Removed for now